test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5067,15 +5066,14 @@ BOOST_AUTO_TEST_CASE(inline_assembly_storage_in_modifiers)
 			modifier m {
 				assembly {
 					x := 2
-					pop
 				}
 				_;
 			}
 			function f() m {
 			}
 		}
 	)";
-	CHECK_ERROR(text, DeclarationError, "");
+	CHECK_ERROR(text, DeclarationError, "Variable not found or variable not lvalue.");
```

Comment:
I'm not sure what you mean by "test case for `{ let y := 3 { 2 { let x := y } pop} pop }`", since that should not be valid inline assembly - the second `pop` has to be removed. Should it be a semantic test or just a parser test?